### ISSUE TITLE
How fine-tune the model?

### ISSUE THREAD

**Issue by SalvatoreRa** [facebookresearch/hiera#12]
Very nice work,

do you have a sample of code for fine-tuning the model for a task?

Could be possible to fine tune as it was a model from the torch hub?

**Comment by dbolya** [comment]
Yes, you should be able to fine-tune a torch hub model like normal.

As for how to fine-tune the models, I'd suggest taking a look at the [finetuning script](https://github.com/facebookresearch/mae/blob/main/main_finetune.py) in the [mae repo](https://github.com/facebookresearch/mae). It should be fairly similar to what we did, but you might have to tweak "drop path" and "layer decay" to your specific task.

We will eventually release training code in this repo as well, but we're currently fairly busy so it might take a while.

**Comment by AdrienBouchet1** [comment]
Will you release an official finetuning script ? I can't find anything so far.
Bests,
